src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -6,18 +6,21 @@ import {
   useStyleConfig,
 } from "@chakra-ui/react";
 import React, { forwardRef } from "react";
+
 import Icon from "../Icons/Icon";
 import Tooltip from "../Tooltip/Tooltip";
 import { truncateText } from "../../utils/utils";
 
-export type BreadcrumbsTypes =
-  | "blogs"
-  | "booksAndMore"
-  | "brand"
-  | "education"
-  | "locations"
-  | "research"
-  | "whatsOn";
+export const breadcrumbTypeArray = [
```

Comment:
`BreadcrumbsTypes` is exported in `index.ts`.  That should be updated.  And I am curious how this type of change will affect the use of that export.

Reply:
It shouldn't affect the export. The type is still exported and is the same as before, but how it was created was updated.